cro-clib/examples/Makefile

```diff
@@ -1,17 +1,22 @@
 all: build run
 build:
+# for rocksdb link errors
+	export AR=/usr/bin/ar
 	cargo build
+	sed   's/Option_ProgressCallback/ProgressCallback/' -i  ../chain.h
```

Comment:
could this be patched in https://github.com/eqrion/cbindgen/blob/master/docs.md#cbindgentoml
there's "rename" or include/exclude for export?

Reply:
i'll check